source/gui/contextHelp.py

```diff
@@ -40,13 +40,7 @@ def showHelp(helpId: str):
 		return
 	helpFile = documentationUtils.getDocFilePath("userGuide.html")
 	if helpFile is None:
-		# Translators: Message shown when trying to display context sensitive help,
-		# indicating that	the user guide could not be found.
-		noHelpMessage = _("No user guide found.")
-		log.debugWarning(
-			"No user guide found: possible cause - running from source without building user docs",
-		)
-		queueHandler.queueFunction(queueHandler.eventQueue, ui.message, noHelpMessage)
+		documentationUtils.reportNoDocumentation("userGuide.html")
```

Comment:
Should this use a message box as well, or would this conflict with context help being requested from inside modal dialogs in NVDA?

Reply:
Currently, we are using `ui.message` for context help.

Taking into account that the current solution is working and that the use case of missing documentation is quite rare (so rarely tested), I have not investigated the message box solution and have opted to keep the same design as today.

Using a message box for context help seems to be working. I am not totally sure if specific precautions should be taken and which specific cases should be tested regarding modal dialogs and restart race conditions. Maybe @seanbudd can help for this?

Reply:
I think it should be fine to stick with `ui.message` in these cases then. If users have problems with the current approach, a new issue can be opened for it and we can revisit using a message box instead.